Makefile

```diff
@@ -62,3 +62,11 @@ include build/makelib/docs.mk
 # Generate manifests e.g. CRD, RBAC etc.
 manifests:
 	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd --output-dir cluster/charts/crossplane/crds --nested
+
+# Generate a coverage report for cobertura applying exclusions on
+# - generated file
+# - runtime scaffolding files
```

Comment:
i see `zz_generated.deepcopy` excluded, which i assume covers the `generated file` clause in the comment.  how are the `runtime scaffolding files` excluded as well?